ast/ast.hpp

```diff
@@ -142,9 +148,9 @@ struct Statement : Node {
 struct Assignment : Statement {
     void accept(Visitor& v) override { v.visit(this); }
 };
-struct RoutineCall : Statement {
-    void accept(Visitor& v) override { v.visit(this); }
-};
+// struct RoutineCall : Statement {
+//     void accept(Visitor& v) override { v.visit(this); }
+// };
```

Comment:
Again, commented here just to be redefined later in the file?

Reply:
I would prefer it if the definitions were in the same order as the declarations

Reply:
As I have already said, I'm trying to preserve the logical order. Like the deeper I use the structure, the deeper it is defined.